OriginalREADME.md

```diff
@@ -0,0 +1,379 @@
+# SignalFx Smart Agent 
+
+[![GoDoc](https://godoc.org/github.com/signalfx/signalfx-agent?status.svg)](https://godoc.org/github.com/signalfx/signalfx-agent)
+[![CircleCI](https://circleci.com/gh/signalfx/signalfx-agent.svg?style=shield)](https://circleci.com/gh/signalfx/signalfx-agent)
+
+The SignalFx Smart Agent is a metric agent written in Go for monitoring
```

Comment:
Confirm that we want to keep the info about the old agent still being used internally.